tests/Unit/DataStructures/DataBox/Test_DataBoxDocumentation.cpp

```diff
@@ -0,0 +1,512 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
```

Comment:
clang format off at beginning of file, on at end

we want the code to be nicely formatted for the documentation, not here